waiter/resources/web/maintenance.txt

```diff
@@ -13,8 +13,7 @@ Request Info
         Method: <%= request-method %>
            CID: <%= cid %>
           Time: <%= timestamp %><% (when principal %>
-     Principal: <%= principal %><% ) %><% (when service-id %>
-    Service Id: <%= service-id %><% ) %>
+     Principal: <%= principal %><% ) %>
```

Comment:
I suspect principal will also be missing?

Reply:
The principal should also be missing
This populates the `authorization/principal` key:
https://github.com/twosigma/waiter/blob/adbabdb86508b973481411a1183f0026a0c2a743/waiter/src/waiter/auth/authentication.clj#L39
https://github.com/twosigma/waiter/blob/adbabdb86508b973481411a1183f0026a0c2a743/waiter/src/waiter/core.clj#L961
https://github.com/twosigma/waiter/blob/adbabdb86508b973481411a1183f0026a0c2a743/waiter/src/waiter/core.clj#L1915
https://github.com/twosigma/waiter/blob/adbabdb86508b973481411a1183f0026a0c2a743/waiter/src/waiter/core.clj#L1550

The `wrap-auth-secure-fn` middleware eventually calls the `auth/wrap-auth-handler` function which populates the principal, and because the middleware is called after the maintenance middleware, `authorization/principal` is not populated when the request is going through the maintenance middleware.

I'll remove principal field from the html and txt